### PR TITLE
Use lib_cellar in python3 for better consistency

### PR DESCRIPTION
**Pull request by yesimon**
Port over the fix from the python2 formula for compatibility with linuxbrew.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
What is this fixing?

**Comment by yesimon** [comment]
Okay, the actual fix is more appropriate in downstream linuxbrew in Homebrew/linuxbrew#329, so I changed this PR into adding a lib_cellar variable to be more consistent with the python2 formula, and also simplifies downstream merging in linuxbrew.

**Comment by mikemcquaid** [comment]
CC @tdsmith for thoughts.

File: Library/Formula/python3.rb [hunk 1]
```diff
@@ -50,8 +50,12 @@ class Python3 < Formula
   # X11.
   patch :DATA if build.with? "brewed-tk"
 
+  def lib_cellar
+    prefix/"Frameworks/Python.framework/Versions/2.7/lib/python2.7"
```
Comment (tdsmith) [inline Library/Formula/python3.rb]: 2.7 is wrong here.

**Comment by tdsmith** [comment]
No other objections.

**Comment by yesimon** [comment]
Fixed.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `python3: use lib_cellar variable for consistency.`.

**Pull request closed without merging**